### PR TITLE
Added 6 resources in various fields

### PR DESCRIPTION
**Pull request by LegendaryGene**
Added 6 crucial resources in EE, ME, ML and CP fields

### CODE REVIEW

**Comment by LegendaryGene** [comment]
Please Merge @aaryan2134

File: README.md [hunk 1]
```diff
@@ -75,6 +75,8 @@ If the PR is not according to the guidelines then it will not be accepted.
 4. [Network Analysis Playlist](https://www.youtube.com/watch?v=FF-rLTSynBQ&list=PL4zFSWGjyVv5uxvzbVLkn40vV42IQYB3y)
 5. [Signals and Systems Playlist](https://www.youtube.com/watch?v=s8rsR_TStaA&list=PLBlnK6fEyqRhG6s3jYIU48CqsT5cyiDTO)
 6. [digital and Analog Playlist](https://www.youtube.com/watch?v=qhjj6WG7Rgc&list=PLwjK_iyK4LLArUHRm3SvPLT0XWlVhpl4h).
+7. [Basic Electronics Course](https://www.youtube.com/watch?v=lFCMYEmfgzU&list=PLTFMW-rP7fuqMiK174et4SKbbMKd4VjdU)
+8. [MicroElectrnics Course](https://www.youtube.com/watch?v=366BVdmcUxk&list=PLLy_2iUCG87C-GS7B4vJ37Iz3fevmYU1h)
```
Comment (aaryan2134) [inline README.md:79]: Please correct the spelling

File: README.md [hunk 2]
```diff
@@ -115,6 +119,8 @@ If the PR is not according to the guidelines then it will not be accepted.
 12. [Segment Tree](https://youtube.com/playlist?list=PL-Jc9J83PIiFY1445K_x1M_O3j9kr3FII).
 12. [CSES Problem Set](https://cses.fi/problemset/).
 12. [Number Theory for Competitive Programming](https://youtube.com/playlist?list=PLauivoElc3giVROwL-6g9hO-LlSen_NaV).
+13. [CP Ladder CodeForces](https://www.google.com/url?sa=t&source=web&rct=j&url=https://earthshakira.github.io/a2oj-clientside/server/Ladders.html&ved=2ahUKEwiw6fOJxeb2AhXXAYgKHUhJCxQQFnoECAQQAQ&usg=AOvVaw06fEjXvZfjfmryW3QzYaeC)
```
Comment (aaryan2134) [inline README.md:122]: Please rectify this link

File: README.md [hunk 3]
```diff
@@ -115,6 +119,8 @@ If the PR is not according to the guidelines then it will not be accepted.
...
+14. [GeeksForGeeks](https://www.geeksforgeeks.org/)
```
Comment (aaryan2134) [inline README.md:123]: This is not an appropriate resource for this section

**Review by aaryan2134 — CHANGES_REQUESTED** [review]
Please make the necessary changes

File: README.md (hunk 2 above)
Comment (LegendaryGene) [reply README.md:122]: This link is working isn't it?

**Comment by LegendaryGene** [comment]
Please review @aaryan2134

**Comment by aaryan2134** [comment]
Please resolve conflicts

**Comment by aaryan2134** [comment]
Otherwise it looks good

**Comment by LegendaryGene** [comment]
Shall I open a new PR @aaryan2134, don't really know how to resolve the issue

**Comment by LegendaryGene** [comment]
> Please resolve conflicts

Done @aaryan2134 , please merge asap, thanks :P

**Comment by aaryan2134** [comment]
Thank you

**Pull request merged**